win-spout.cpp

```diff
@@ -186,13 +234,17 @@ static void win_spout_deinit(void *data)
 		context->texture = NULL;
 	}
 	// cleanup spout
-	context->spoutptr->ReleaseReceiver();
+	if (context->need_release) {
+		context->spoutptr->ReleaseReceiver();
+		context->need_release = false;
```

Comment:
```suggestion
		context->should_release = false;
```

Reply:
https://github.com/leadedge/Spout2/blob/851deec62ef8f1e7b90c0df1448387e0d2841de1/SpoutSDK/Source/SpoutSDK.cpp#L320 
```
	Sleep(100); // Debugging aid, but leave for safety
```
what the hell?

Reply:
🤯 - who knows why?

Reply:
For us it means, each time we call releaseReceiver, OBS must skip 3 frame(30fps) . Is it safe to only call releaseReceiver at destory ?

Reply:
And i also noticed if the sender accidentally exit , it will left a sender have name but no info until any sender use this same name.